### ISSUE TITLE
Suggestion: Change 'pin' in upgrade list

### ISSUE THREAD

**Issue by paul59** [tobspr/shapez.io#1119]
New player perspective: the pin symbol in the 'Upgrade' window is counterintuitive, surely the pin should be lit/brighter when activated? Maybe it would be clearer if it was a checkbox or a pin illuminated in green if active (although I appreciate that might cause issues for the colour-blind).

**Comment by EmeraldBlock** [comment]
I interpret as the pin being gray as it being "activated", but I can definitely see people having either interpretation

**Comment by paul59** [comment]
> I interpret as the pin being gray as it being "activated", but I can definitely see people having either interpretation

What about the sliders in 'Options', grey when inactive/false?

**Comment by EmeraldBlock** [comment]
Those are blue / gray as opposed to gray / white, though I get what you mean

**Comment by paul59** [comment]
> Those are blue / gray as opposed to gray / white, though I get what you mean

Yeah that would work for consistency at least (and less to code/change :D) : blue when pin active, grey when not.

**Comment by joachim-n** [comment]
I also found the colour of the pin icon the opposite way of what I expected.
